sympy/polys/tests/test_rootoftools.py

```diff
@@ -566,3 +565,9 @@ def test_eval_approx_relative():
     assert [str(i) for i in a] == [
         '-0.10', '0.05 - 3.2*I', '0.05 + 3.2*I']
     assert all(abs(((a[i] - t[i])/t[i]).n()) < 1e-2 for i in range(len(a)))
+
+def test_issue_15920():
+    x = Symbol("x")
```

Comment:
not needed

please put space around `-+` operators (below) when fixing

Reply:
I patched it up. This should be ready to go.

Reply:
thanks